app/celer-sim/simple-driver.py

```diff
@@ -141,6 +142,12 @@ def strtobool(text):
     json.dump(j, f, indent=1)
 print("Results written to", outfilename, file=stderr)
 
-time = j['result']['runner']['time'].copy()
-time.pop('steps')
+run_output =j['result']['runner']
+time = run_output['time'].copy()
+steps = time.pop('steps')
+if use_device:
+    assert(len(steps) == run_output['num_step_iterations'][0])
```

Comment:
Shouldn't it be `len(steps[0])` since there's an array of step times for each event?

Reply:
Damn it why didn't I just hit "commit suggestion" 😩